rollup.config.js

```diff
@@ -65,10 +84,12 @@ const plugins = [
   json(),
 ]
 
-if (prod) plugins.push(uglify(), visualizer({ filename: './bundle-stats.html' }))
+if (prod) {
+  if (noParser) { plugins.push(uglify(), visualizer({ filename: './bundle-stats-no-parser.html' })) } else { plugins.push(uglify(), visualizer({ filename: './bundle-stats.html' })) }
+}
 
 export default {
-  entry: 'src/index.js',
```

Comment:
Can you put the entry ternary closer to here or inline? Easier to read i.e. The reader doesn't have to scroll up